pkg/apiserver/resthandler.go

```diff
@@ -299,6 +301,27 @@ func createHandler(r rest.NamedCreater, scope RequestScope, typer runtime.Object
 			return
 		}
 
+		unstruct := &runtime.Unstructured{}
+		err = runtime.UnstructuredJSONScheme.DecodeInto(body, unstruct)
```

Comment:
Perhaps we could refactor Scheme.Decode to make that possible?

See also #10234